### PR TITLE
fruit reaction field

### PR DESCRIPTION
**Pull request by mcfiredrill**
This will be used as part of the fruit reaction field implementation.

### CODE REVIEW

**Comment by mcfiredrill** [comment]
This was a WIP but its turning into a viable implementation.

Still want to check that this doesn't totally break any browsers if they dont support webGL.

hotdog lounge PR is already merged and deployed: https://github.com/datafruits/hotdog_lounge/pull/17

**Comment by mcfiredrill** [comment]
@dudethatbe both had problems getting this to work in Firefox on Linux.

Looks like the canvas fallback is no longer part of pixi.js.

```
Uncaught Error: WebGL unsupported in this browser, use "pixi.js-legacy" for fallback canvas2d support.
    create Renderer.js:50
    Tn autoDetectRenderer.js:36
    t Application.js:67
    <anonymous> script.js:11
Renderer.js:50:14
    create Renderer.js:50
    Tn autoDetectRenderer.js:36
    t Application.js:67
    <anonymous> script.js:11```

**Comment by mcfiredrill** [comment]
## TODO

- [ ] handle window resizing, resize the canvas
- [ ] improve disappearing animation

**Comment by mcfiredrill** [comment]
Here's a little demo if you haven't seen it yet.

https://user-images.githubusercontent.com/66243/105651377-07c33280-5efa-11eb-8d81-7266e5d97ff0.mp4

**Comment by mcfiredrill** [comment]
> @dudethatbe both had problems getting this to work in Firefox on Linux.

Seems like this was because neither of us had webGL working in our Firefox installs. 
I didn't really try setting up pixi.js-legacy yet, but it might be a simple drop in replacement? https://www.npmjs.com/package/pixi.js-legacy

File: app/components/fruit-tip.hbs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      </button>
+    {{/each}}
+  </div>
+{{/if}}
+<button
+  {{on "click" this.showFruitChoices}}
+  class="cool-button fruit-tip-button"
+  type="button"
+  >
+  give the DJ fruit
```
Comment (dudethatbe) [inline app/components/fruit-tip.hbs:26]: was thinking this needed to be in the translations, but what would be the best way of phrasing this across all of the locales? Maybe the button text could be a fruit emoji and the title is "give the DJ fruit".

**Review by dudethatbe — APPROVED** [review]
Aside from the issues noted earlier, I think its great. The wording of the button itself can be improved for translations of course, and there's some blocks of comments that I'm not sure should still be there or not. I'm sure everyone's gonna love the new feature :smile:

File: app/components/fruit-tip.hbs (hunk 1 above)
Comment (mcfiredrill) [reply app/components/fruit-tip.hbs:26]: Yes I think emoji with the text in the mouse hover title would be much better :strawberry:

**Pull request merged**